service/patch.go

```diff
@@ -79,15 +80,27 @@ func (uis *UIServer) patchPage(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	commitQueuePosition := 0
+	if projCtx.Patch.CommitQueue {
+		cq, err := commitqueue.FindOneId(project.Identifier)
```

Comment:
an alternative to passing the position together with the rest of the patch page is for the page to use the existing rest route to list the items in the queue: GET "projects/{project_id}/commit_queue"

Reply:
unless there's a specific reason to use the route, I think this direct computation is maybe better (and the position would still have to be passed with the rest of the patch page I imagine).

Reply:
I don't see a reason to use the route.